### ISSUE TITLE
`@stylistic/function-parentheses-newline-inside` empties only part of the whitespace it measured and reports the fix as written

### ISSUE THREAD

**Issue by firefoxic** [stylelint-stylistic/stylelint-stylistic#285]
Found while fixing #281, by checking what the guard that issue asks for lets through. It lets this through rightly — the comma stands outside the comment on either side of the fix — and the fix still cannot satisfy the option, because the walk that measures the whitespace reaches further than the fix that empties it. `main` behaves the same way, so this is older than that branch.

### What minimal example or steps are needed to reproduce the bug?

```less
a { t: translate( // c
 /*x*/
 , 1px); }
```

### What minimal configuration is needed to reproduce the bug?

```json
{
	"customSyntax": "postcss-less",
	"plugins": "@stylistic/stylelint-plugin",
	"rules": {
		"@stylistic/function-parentheses-newline-inside": "never-multi-line"
	}
}
```

### What did you expect to happen?

Either the whitespace behind `translate(` gone, or the problem reported and nothing written. Those are the two answers every guard of this family gives.

### What actually happened?

The space in front of the comment is taken away and the line break behind the comment is left standing:

```less
a { t: translate(// c
 /*x*/
 , 1px); }
```

The option is still violated, `--fix` reports a clean pass, and the next run reports the same problem again. The testing library has caught this since 1.1.1 — it relints what a fixer wrote — so any case written for this shape fails on the assertion that a fixer may have suppressed a warning without resolving it.

### Do you have a proposal to fix the bug?

The two walks disagree about one thing. `getCheckBefore` counts `valueNode.before`, every space node in front of the first significant node, and the whitespace a node overruns the end of an inline comment's span by; it steps over the nodes such a comment is spelled with, since `postcss-value-parser` knows nothing of one and hands out two division signs and a word where one stands. `fixBeforeForNever` stops at the first node that is neither a comment nor a space, which is the first of those division signs. So wherever an inline comment stands between the parenthesis and the first argument, the measurement reaches past it and the fix does not.

Either walk can be brought to the other, and the choice is what wants deciding:

- teaching the fix the same walk empties everything measured, and what it closes up against the comment is comments and nothing else — the first significant node is what the guard of #281 stands in front of;
- teaching the guard to decline wherever the fix cannot empty everything counted reports and writes nothing, which is what this rule does whenever the option cannot be satisfied.

`function-parentheses-space-inside` measures its `before` differently and is not this; #280 is the guard it has yet to grow.
